src/lib.rs

```diff
@@ -162,46 +238,74 @@ pub enum RestartOptions {
 }
 
 #[derive(Clone)]
-pub struct Options {
+struct Options {
     restart: RestartOptions,
     verbose: bool,
     file_handle_flags: bool,
 }
 
-impl Options {
-    pub fn new() -> Options {
-        Options {
+pub struct Runner<C: CommandLike> {
+    commands: Vec<C>,
+    restart: RestartOptions,
+    verbose: bool,
+    file_handle_flags: bool,
+}
+
+impl<C: CommandLike + Clone> Default for Runner<C> {
+    fn default() -> Self {
+        Runner::new()
+    }
+}
```

Comment:
Why is `Default` only implemented for `CommandLike + Clone` here? I don't see any cloning going on :slightly_smiling_face:

Reply:
this is clippy wanting an impl for Default for Runner. Runner requires both CommandLIke and Clone

Reply:
I moved Clone to be a supertrait of CommandLike to clean up the syntax of Clone everywhere.

Overall, clone is added so commands can be passed as refs (pointers) or structures themselves while still being safe to pass across thread boundaries